gemrb/GUIScripts/LevelUp.py

```diff
@@ -159,20 +159,23 @@ def OpenLevelUpWindow():
 	if not IsMulti:
 		# check if we're dual classed
 		IsDual = GUICommon.IsDualClassed (pc, 1)
-		Classes = [IsDual[2], IsDual[1]] # make sure the new class is first
+		Classes = []
 
 		# either dual or single only care about 1 class
 		NumClasses = 1
 
 		# not dual, must be single
 		if IsDual[0] == 0:
 			Classes = [Class]
-		else: # make sure Classes[1] is a class, not a kit
-			if IsDual[0] == 1: # kit
-				Classes[1] = CommonTables.KitList.GetValue (IsDual[1], 7)
-			else: # class
-				TmpClassName = GUICommon.GetClassRowName (Classes[1], "index")
-				Classes[1] = CommonTables.Classes.GetValue (TmpClassName, "ID")
+		else: # resolve kits to classes (new class goes first)
```

Comment:
why the reverse order?

Reply:
I don't even know!  I was just parroting the line 162 above.